src/inverted-index.js

```diff
@@ -22,31 +22,26 @@ class Index {
     var self = this;
 
     return this.readJson(url).then(function(response) {
-      var data = response;
       //get the documents in the json object and store the content in an array
-      data.map(function(element, index) {
+      response.map(function(element, index) {
         var content = [];
         var titles = Object.keys(element);
         var i = 0;
 
-        for (i = 0; i < titles.length; i++) {
-          content.push(element[titles[i]]);
-        }
-        //split the content into words and store each word in the invertedIndex
```

Comment:
**Code Quality - Nitpick**
- Redundant array mapping. Find the simplest and most efficient logic of executing the functionalities.